nixos/modules/programs/shell.nix

```diff
@@ -8,12 +8,6 @@ with lib;
 
   config = {
 
-    environment.shellAliases =
-      { ls = "ls --color=tty";
-        ll = "ls -l";
-        l  = "ls -alh";
-      };
-
```

Comment:
Not entirely sure about removing those, they have been there for a long time. I don't personally use them, but maybe some people do. Do other distros have any default aliases?

Also, the cost of keeping these around with the changes in this PR isn't very high, so we might as well just leave them in for the sake of not annoying anybody.

Reply:
Okey. Moved them to the default of `environment.shellAliases`.